docs/configs/site.js

```diff
@@ -30,7 +30,7 @@ export default {
   'en-us': {
     banner: {
       text: '🤔 Have queries regarding Apache DolphinScheduler, Join Slack channel to disscuss them ',
-      link: 'https://join.slack.com/t/asf-dolphinscheduler/shared_invite/zt-1e0s4agia-Ofkxa93zStmqTm9UsQBE8g'
+      link: 'https://join.slack.com/t/asf-dolphinscheduler/shared_invite/zt-1e36toy4n-5n9U2R__FDM05R~MJFFVBg'
```

Comment:
Why we need to update the slack URL? I find the before link can work well.

Reply:
> Why we need to update the slack URL? I find the before link can work well.

@labbomb I have the same question, is the invitation url no valid?

Reply:
PTAL @nielifeng

Reply:
Is there a 30-day expiration for the invitation link?

Reply:
Change account and regenerate link

Reply:
So if it doesn't seem to be a problem, this PR is merged, right